src/main/java/pene/gc/importer/commands/Import.java

```diff
@@ -67,12 +67,19 @@ public final class Import implements CommandHandler {
                 CommandHandler.sendMessage(targetPlayer,"Artifacts import disabled");
             }
         }
+
+        if (GenshinImporter.getPluginConfig().Weapons) {
+            Datareader.weapons(targetPlayer, filename);
```

Comment:
Also you forgot to add weapons function to datareader

Reply:
Whoops, seems that I've missed that when making commits to Git :(